### ISSUE TITLE
what are the "-----"

### ISSUE THREAD

**Issue by jeffchen2000** [cbg-ethz/shorah#56]
in global haplotype sequence as below, I see "----", are they sequence gaps/deletions? or unclear bases like 'N'?

GAATACCCGGCAGATTACTTCAGAAAATCAAAGGAGATTCCTCTTTACATCAATACTA--
-------TGTCAGATCTAAGAGGATATGTCTACCAAGGCCTCAAATCCGGAAATGTATCA
ATCATACATGTCAACAGCTACTTGTATGGAGCATTAAAGGACATCCGGGGTAAGTTGGAT

**Comment by DrYak** [comment]
> I see "----", are they sequence gaps/deletions?

yes, exactly. at this point the reconstructed haplotype has gaps in regards to the reference.
